website/search/views.py

```diff
@@ -222,3 +228,44 @@ def search_share_stats():
     query = build_query(q, 0, 0) if q else {}
 
     return search.share_stats(query=query)
+
+
+def search_share_atom(**kwargs):
+    q = request.args.get('q', '*')
+    sort = request.args.get('sort', 'dateUpdated')
+
+    # we want the size to be constant between pages
+    # TODO -  move this functionality into build_query in util
+    size = 250
```

Comment:
Could you move this definition to the top of the file and make it all uppercase with a slightly more descriptive name?